core/app/models/spree/order.rb

```diff
@@ -400,7 +400,7 @@ def deliver_order_confirmation_email
 
     # Helper methods for checkout steps
     def paid?
-      payment_state == 'paid' || payment_state == 'credit_owed'
+      %w(paid credit_owed).include?(payment_state)
```

Comment:
> My preference would be we let things settle for a few more days/weeks before we global change stuff.

So why do you accept any PRs right now, before this happened? Just want to understand the global plan here. Right now it does seem quite arbitrary, but thats from someone who does not know the global plan.